hera_cal/lst_stack/binning.py

```diff
@@ -246,14 +260,33 @@ def get_lst_bins(
     return bins, lsts, mask
 
 
+def _get_freqs_chans(freqs, freq_min, freq_max):
+
+    if freq_min is None and freq_max is None:
+        freq_chans = None
+    else:
+        freq_chans = np.arange(len(freqs))
+
+    if freq_min is not None:
+        mask = freqs >= freq_min
+        freqs = freqs[mask]
+        freq_chans = freq_chans[mask]
+    if freq_max is not None:
+        mask = freqs <= freq_max
+        freqs = freqs[mask]
+        freq_chans = freq_chans[mask]
+
+    return freqs, freq_chans
+
+
 def lst_bin_files_for_baselines(
     data_files: list[Path | FastUVH5Meta],
     lst_bin_edges: np.ndarray,
     antpairs: Sequence[tuple[int, int]],
     freqs: np.ndarray | None = None,
     pols: np.ndarray | None = None,
     cal_files: list[Path | None] | None = None,
-    time_arrays: list[np.ndarray] | None = None,
+    # time_arrays: list[np.ndarray] | None = None,
```

Comment:
Should probably delete this line and its mention in the documentation if not being used